tests/integration-tests/tests/api_infrastructure/test_api_infrastructure.py

```diff
@@ -0,0 +1,131 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+import os
+import re
+
+import boto3
+import pytest
+from assertpy import assert_that, soft_assertions
+from cfn_stacks_factory import CfnStack, CfnStacksFactory
+from utils import generate_stack_name
+
+
+@pytest.fixture()
+@pytest.mark.usefixtures("setup_sts_credentials")
+def api_infrastructure_stack_factory(request):
+    """Define a fixture to manage the creation and destruction of the API Infrastructure CloudFormation stack."""
+    factory = CfnStacksFactory(request.config.getoption("credential"))
+
+    def _create_api_infrastructure(region, parameters):
+        template_path = os.path.join("..", "..", "api", "infrastructure", "parallelcluster-api.yaml")
+        file_content = extract_template(template_path)
+        stack = CfnStack(
+            name=generate_stack_name("integ-tests-api-infrastructure", request.config.getoption("stackname_suffix")),
+            region=region,
+            template=file_content,
+            parameters=parameters,
+            capabilities=["CAPABILITY_AUTO_EXPAND", "CAPABILITY_IAM"],
+        )
+        factory.create_stack(stack)
+        return stack
+
+    def extract_template(template_path):
+        with open(template_path) as cfn_file:
+            file_content = cfn_file.read()
+        return file_content
+
+    yield _create_api_infrastructure
+    factory.delete_all_stacks()
+
+
+@pytest.mark.skip_regions(["cn-north-1", "cn-northwest-1"])  # TODO No Lambda container support in China regions
+@pytest.mark.usefixtures("os", "instance", "scheduler")
```

Comment:
I think you can get rid of this one if you don't need to parametrize on these dimensions

Reply:
Are you referring to `@pytest.mark.usefixtures("os", "instance", "scheduler")`? I don't know if I have interpreted wrongly [what's written in the README](https://github.com/ermanno/aws-parallelcluster/blob/wip/pcluster3/tests/integration-tests/README.md#define-parametrized-test-cases), but I was under the impression that the fixture was a way to "comply" with the test framework in case we didn't really need to access the listed dimensions

Reply:
Confirmed that the test can be run without the fixture, will proceed to remove it